.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
user-specific (rather than project specific) things can be ignored locally:
```sh
cat > "$HOME/.config/git/gitignore" <<EOF
.idea/
EOF
git config --global core.excludesfile "$HOME/.config/git/gitignore"
```

Reply:
Thanks, removed!